src/webui/webapplication.h

```diff
@@ -54,7 +54,7 @@
 #include "base/utils/version.h"
 #include "api/isessionmanager.h"
 
-inline const Utils::Version<3, 2> API_VERSION {2, 11, 0};
+inline const Utils::Version<3, 2> API_VERSION {2, 12, 0};
```

Comment:
@Chocobo1
Shouldn't it be the following instead?
```suggestion
inline const Utils::Version<3, 2> API_VERSION {2, 11, 1};
```
Now it doesn't introduce breaking changes, does it?

Reply:
yes, it should.